src/Action/EntrypointAction.php

```diff
@@ -9,12 +9,12 @@
  * file that was distributed with this source code.
  */
 
-namespace ApiPlatform\Core\Hydra\Action;
+namespace ApiPlatform\Core\Action;
```

Comment:
IMO we can get rid of the `Hypermedia` namespace. The entrypoint can be useful even in a non-hypermedia context (i.e raw JSON or XML).